### ISSUE TITLE
developer meeting experiment

### ISSUE THREAD

**Issue by clux** [kube-rs/kube#1319]
One of the few pieces of feedback that I personally think is sensible to take on-board from CNCF as suggestions, is having some form of less formal means of real-time communication.

We've never had this before, and we get by, but a lot of the more difficult issues on here can sometimes stagnate when:

1. collective maintainer bandwidth is limited (and we prefer solving the easy issue/qa of the week)
2. it can be hard to engage with issues that we don't feel like we have the full context of

And i think it's valid to be able to talk about issues and direction in a less formal and less daunting way than github issues, to get pointers on where to start and to maybe seek out an exploratory consensus without writing an RFC equivalent.

I also do think we have a lot of (potentially deteriorating) knowledge and assumptions on "what we'd like things to be like" that we don't always express because it's a lot of effort to write it all down and justify formally. Particularly since this is basically volunteer work.

### Proposal
**a meeting, biweekly, with minutes, google doc for next session's agenda that we all can plop things into**. running it for a 3mo period as an experiment.

**Intent**: for **dev related work** and directional planning / exploration, not for issue Q/A ideally (there's enough channels for that and they benefit from searchability) - but some of that is also probably unavoidable.

basically; an optional maintainer-run thing for when time allows (if no-one can / or nothing in the agenda we cancel). I am happy to try to run during an initial experiment (with the caveat that it has to be during EU daytime for a while for me for a while).

**would any one of you find that valuable, or would you consider joining this at times?**

> tagging @kube-rs/maintainers and @kube-rs/triagewrite to query interest
> to be clear: none of you should feel compelled to do this. feel free to 👍 or 👎 without ceremony.

if you do find this valuable, please say so. will try to schedule based on our preferences or time constraints, but ideally we leave a resulting meeting open so that other people wanting to contribute / join in can.

**Comment by nightkr** [comment]
Didn't see this (sorry!) but it seems like a good idea to me.

**Comment by soenkeliebau** [comment]
I concur, sounds like a good thing to try!

**Comment by Dav1dde** [comment]
👍

**Comment by clux** [comment]
Have setup a [google doc for meeting tracking](https://docs.google.com/document/d/1b7OXW0XYREpyq_9xqpRVCDML1k1NNiKTDZqY-DIM6Ig/edit) it should link to a [calendar event](https://www.google.com/calendar/event?eid=MjgxbGdocW1hYnNoZ2drbTBkaHNoMXRjcGZfMjAyMzEyMDVUMTMwMDAwWiBzc3p5bnJhZUBt) that you should be able to join in. Hope google stack is ok for this.

Have copied a template somewhere so far so there's access for people to add in stuff.

LMK access troubles / hard maintainer time conflicts (if you want to join). Plan on moving it a bit later come January anyway though.

**Comment by clux** [comment]
Supplying a calendar link seems to be a better way; so here is the [kube-rs calendar](https://discord.com/channels/500028886025895936/1176778619469705236/1177211849679327262).

**Comment by clux** [comment]
Based on requests have moved this to Thursdays same time.

**Comment by clux** [comment]
This experiment has not been hugely successful, so I am going to close the meeting after having consistently cancelled it in the last 6 months.

Marking it as cancelled in the [agenda doc](https://docs.google.com/document/d/1b7OXW0XYREpyq_9xqpRVCDML1k1NNiKTDZqY-DIM6Ig).

I am still open to collaborating with people over video for things here, but most of the time this meeting ended up being small discussions with maintainers (who are all volunteers with other commitments), so maybe a scheduled setup does not make that much sense to us. Maybe it's a barrier to entry thing, but with this low usage, it's honestly easier to ask people in discord first and then - if necessary - maybe schedule something (as i have been doing a handful of times).

At any rate, will leave it to async discord + github discussions for now.
